src/controllers/currentAccount.ts

```diff
@@ -325,4 +325,47 @@ export class CurrentAccountController {
       [`balance_${address}`]: balance,
     });
   }
+
+  async updateOtherAccountsBalances() {
+    const currentNetwork = this.getNetwork();
+    const url = new URL('addresses/balance', this.getNode());
+    const addresses = this.getAccounts().map(account => account.address);
+
+    while (addresses.length > 0) {
+      const splicedAddresses = addresses.splice(0, 1000);
+      const response = await fetch(url, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
```

Comment:
It's better to add `accept` header as well, so fields with 64-bit integers will be returned as strings instead of numbers:

```js
accept: 'application/json; large-significand-format=string',
```

There are examples of this right in this file.
https://docs.waves.tech/en/waves-node/node-api/headers